src/com/carolinarollergirls/scoreboard/core/impl/ScoreBoardImpl.java

```diff
@@ -291,7 +292,10 @@ public void stopJamTO() {
                 setLabels(ACTION_START_JAM, ACTION_NONE, ACTION_TIMEOUT);
                 _endTimeout(false);
                 finishReplace();
-            } else if (!lc.isRunning()) {
+            } else if (!lc.isRunning() && (!ic.isRunning() || ic.getTimeElapsed() > 1000L)) {
```

Comment:
I think you meant `<`

Reply:
No. This is the condition under which we process the input.

Unit tests that show it works as intended are in.